fuel/transformers/__init__.py

```diff
@@ -8,12 +8,19 @@
 
 from fuel import config
 from fuel.streams import AbstractDataStream
+from fuel.schemes import BatchSizeScheme
 
 
 @add_metaclass(ABCMeta)
 class Transformer(AbstractDataStream):
     """A data stream that wraps another data stream.
 
+    Subclasses must define a `transform_batch` method (to act on batches),
+    a `transform_example` method (to act on individual examples), or
+    both methods. If a `transform_any` method is defined, it is used
+    for both example and batch inputs. This is useful if the example
+    and batch version of a transformation are the same.
```

Comment:
Maybe say something about how if you are going from batches to examples or examples to batches, you should override `get_data` instead.